CHANGELOG.md

```diff
@@ -8,12 +8,12 @@ MOVE THIS SECTION TO THE CORRECT LOCATION
 ## Unreleased
 
 ### Note for Xcode 12 support
-- All SDKs have been updated to support minimum iOS version 9.0, dropping support for iOS 8.0 in this release. Xcode 12 has dropped support for iOS 8 and requires upgrading to this version to continue building on Xcode 12.
+- All SDKs have been updated to support minimum iOS version 9.0, dropping support for iOS 8.0 in this release. Xcode 12 has dropped support for iOS 8 and requires upgrading to this version to continue building on Xcode 12. ([PR #2981](https://github.com/aws-amplify/aws-sdk-ios/pull/2981))
```

Comment:
s/upgrading to this version/setting your app's minimum supported version to 9.0 or greater/

Reply:
```suggestion
- All SDKs have been updated to support minimum iOS version 9.0, dropping support for iOS 8.0 in this release. Xcode 12 has dropped support for iOS 8 and requires setting your app's minimum supported version to 9.0 or greater to continue building on Xcode 12. ([PR #2981](https://github.com/aws-amplify/aws-sdk-ios/pull/2981))
```